Libraries/LibWeb/CSS/Parser/PropertyParsing.cpp

```diff
@@ -4201,17 +4204,108 @@ RefPtr<StyleValue const> Parser::parse_position_anchor_value(TokenStream<Compone
     return CustomIdentStyleValue::create(*dashed_ident);
 }
 
-// https://drafts.csswg.org/css-anchor-position/#position-area
-RefPtr<StyleValue const> Parser::parse_position_area_value(TokenStream<ComponentValue>& tokens)
+// https://drafts.csswg.org/css-anchor-position/#position-try-fallbacks
+RefPtr<StyleValue const> Parser::parse_position_try_fallbacks_value(TokenStream<ComponentValue>& tokens)
 {
-    // none | <position-area>
+    // none | [ [<dashed-ident> || <try-tactic>] | <'position-area'> ]#
 
-    // none
-    if (auto none_keyword_value = parse_all_as_single_keyword_value(tokens, Keyword::None))
-        return none_keyword_value;
+    if (auto none = parse_all_as_single_keyword_value(tokens, Keyword::None))
+        return none;
 
-    // <position-area>
-    // https://drafts.csswg.org/css-anchor-position/#position-area-syntax
+    return parse_comma_separated_value_list(tokens, [&](auto& inner_tokens) {
+        return parse_single_position_try_fallbacks_value(inner_tokens);
+    });
+}
+
+RefPtr<StyleValue const> Parser::parse_single_position_try_fallbacks_value(TokenStream<ComponentValue>& tokens)
+{
+    // [<dashed-ident> || <try-tactic>] | <'position-area'>
+    auto transaction = tokens.begin_transaction();
+    tokens.discard_whitespace();
+
+    if (auto position_area = parse_position_area(tokens)) {
+        transaction.commit();
+        return position_area;
+    }
+
+    Optional<FlyString> dashed_ident;
+    RefPtr<StyleValue const> try_tactic;
+    while (tokens.has_next_token()) {
+        if (auto try_tactic_value = parse_try_tactic_value(tokens)) {
+            if (try_tactic)
+                return {};
+            try_tactic = try_tactic_value.release_nonnull();
+        } else if (auto maybe_dashed_ident = parse_dashed_ident(tokens); maybe_dashed_ident.has_value()) {
+            if (dashed_ident.has_value())
+                return {};
+            dashed_ident = maybe_dashed_ident.release_value();
+        } else {
+            break;
+        }
+        tokens.discard_whitespace();
+    }
+
+    StyleValueVector values;
+    if (dashed_ident.has_value())
+        values.append(CustomIdentStyleValue::create(dashed_ident.release_value()));
+    if (try_tactic)
+        values.append(try_tactic.release_nonnull());
```

Comment:
Are we supposed to modify the order at parse-time? That sounds like a "do it at computed value time" thing but I'm not sure.